In [30]:
import numpy as np

Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

по формуле для доверительного интервала M-Ty*(Sigma/sqrt(n)) и M+Ty*(Sigma/sqrt(n)) найдем доверительный интервал

In [31]:
80-1.96*(16/256**0.5)

78.04

In [32]:
80+1.96*(16/256**0.5)

81.96

Ответ (78.04;81.96)

В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

найдем мат ожидание  сумма опытаных данных/ 10

In [35]:
(6.9+6.1+ 6.2+ 6.8+ 7.5+ 6.3+ 6.4+ 6.9+ 6.7+ 6.1)/10

6.589999999999999

найдем ско 

In [36]:
x = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
np.std(x)

0.42766809560686203

In [37]:
6.589999999999999- 2.262*(0.42766809560686203/10**0.5)

6.284085929123878

In [38]:
6.589999999999999+ 2.262*(0.42766809560686203/10**0.5)

6.8959140708761195

Ответ: (6.28;6.90)

Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

Нулевая гипотеза:µ = 17 . Альтернативная гипотеза (односторонняя) µ>17. Вычислим наблюдаемое значение критерия:

In [39]:
((17.5 - 17)/4)*100**0.5

1.25

По таблице Лапласа найдем значение (1-2*0.05)/2 = 1,645

Так как 1.25<1.645 то можно считать что средний диаметр 17мм

Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [40]:
x =np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
M=x.mean()
M

198.5

In [41]:
x.var(ddof=1)**0.5


4.453463071962462

α=0,01 из условия задачи

In [42]:
((198.5 - 200)/4.45)*10**0.5

-1.0659362899443976

Из таблицы стьюдента найдем значение при k=9 и α=0.01 = 3.25

Ответ:Гипотеза верна т.к. -3.25<-1.065<3.25